src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
 // write code here
+
+const tree = document.querySelector('.tree');
+
+treeRecursion([tree][0]);
+
+
+
+function treeRecursion(elem) {
+
+  if (elem.children.length !== 0) {
+
+    if (elem.tagName === 'LI') {
+
+      const oldString = elem.firstChild.data.trim()
+      const temp = `<span>${oldString}</span>`;
+
+      const oldString2 = elem.innerHTML.replace(oldString, temp);
```

Comment:
oldString, oldString2, temp  are not really good names :) try to come up with something more meaningful